store.css

```diff
@@ -0,0 +1,109 @@
+/* main section */
+h1 {
+    color: rgb(224, 99, 58);
+    font-weight: 500;
+    margin-top: 50px;
+    text-align: left;
+}
+.logo {
+    height: 5px;
+    width: 5px;
+    float: left;
+    position: fixed;
+    top: 20px;
+    left: 80px;
+}
+.aside_image {
+    width: 800px;
+    height:900px;
+}
+fieldset {
+    border:none;
+    margin-top: 1rem;
+    padding-left: 0;
+}
+/* form section */
+.main_store {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-top: 40px;
+}
+.form {
+    text-align:left;
+    color: grey-light;
+    width: auto;
+    margin:auto;
+}
+.form_title {
+    font-size: 3.5rem;
+}
+/* wraps */
+.wrap_rows {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin:auto;  
+}
+.wrap_column {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+/* inputs */
+.first_name { 
+    margin-bottom: 20px;
+    
+}
+#first_name {
+    margin-top: 20px;
+}
+.last_name {
+    margin-bottom: 20px;
+    width:200px;   
+}
+#lastname{
+    margin-top:20px ;
+    margin-right: 1rem;
+}
+.input_city {
+    margin-top: 20px;
+}
+#postcode {
+    margin-top: 20px;
+}
+.legend {
+    font-size: 1.3rem;
+    padding:2 px;
+    margin-bottom: 1rem;
```

Comment:
Try to use more `relative` units such as `rem` or `em`, rather than absolute units such as `px`. This is considered good practise. 

I think it is considered good practise because when you use your website on different devices, relative units tend to perform better. (_This is my assumption. I could be wrong._)